packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -56,6 +65,22 @@ export class RichTextEditor extends FoundationElement {
 
     private tiptapEditor!: Editor;
 
+    private readonly markdownParser: MarkdownParser;
+    private readonly markdownSerializer: MarkdownSerializer;
+    private readonly domSerializer: DOMSerializer;
+    private readonly xmlSerializer: XMLSerializer;
+
+    /**
+     * @internal
+     */
+    public constructor() {
+        super();
+        this.domSerializer = DOMSerializer.fromSchema(schema);
+        this.xmlSerializer = new XMLSerializer();
+        this.markdownParser = this.initializeMarkdownParser();
+        this.markdownSerializer = this.initializeMarkdownSerializer();
```

Comment:
I would do these initializations in the declarations above, so you don't need a constructor.

Reply:
Initialized in the declarations and removed the constructor.